docs/openapi_3.md

```diff
@@ -0,0 +1,113 @@
+# OpenAPI 3.0/3.1 Support
```

Comment:
would be nice to have this in a more popular place or at least a link in the README